lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2453,16 +2401,44 @@ export class UserAgentApplication {
   /**
    * Set extraQueryParameters to be sent to the server
    * @param extraQueryParameters
-   * @param authenticationRequest
    */
-  private buildServerRequestExtraQueryParameters(extraQueryParameters: KeyValue): string {
-    if (extraQueryParameters) {
-      const paramsStr: string = Utils.generateExtraQueryParameters(extraQueryParameters);
+  private buildServerRequestQueryParameters(params: QPDict): string {
```

Comment:
avoid this method, instead call utils.generateQueryParameters directly?

Reply:
Changes made.